firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/type/Content.kt

```diff
@@ -55,6 +55,9 @@ class Content @JvmOverloads constructor(val role: String? = "user", val parts: L
     /** Wraps the provided [image] inside an [ImagePart] and adds it to the [parts] list. */
     @JvmName("addImage") fun image(image: Bitmap) = part(ImagePart(image))
 
+    @JvmName("addFileData")
```

Comment:
There's no documentation here about what the params are and that files need to be in GCS with adequate perms.